libraries/AP_Soaring/AP_Soaring.cpp

```diff
@@ -142,16 +142,16 @@ const AP_Param::GroupInfo SoaringController::var_info[] = {
     // 19 reserved for POLAR_LEARN.
 
     // @Param: THML_ARSPD
-    // @DisplayName: Specific setting for airspeed when thermalling.
-    // @Description: If non-zero this airspeed will be used when thermalling.
-    // @Values: 0: Use TRIM_ARSPD_CM, >0: Target airspeed in m/s
+    // @DisplayName: Specific setting for airspeed when in a thermal in SOARING.
+    // @Description: If non-zero this airspeed will be used when thermalling. A value of 0 will use TRIM_ARSPD_CM.
+    // @Range: 0 50
     // @User: Advanced
     AP_GROUPINFO("THML_ARSPD", 20, SoaringController, soar_thermal_airspeed, 0),
 
     // @Param: CRSE_ARSPD
-    // @DisplayName: Specific setting for airspeed when cruising in AUTO.
-    // @Description: If non-zero this airspeed will be used when cruising in AUTO. If set to -1, airspeed will be selected based on speed-to-fly theory.
-    // @Values: -1: Use speed-to-fly computed value, 0: Use TRIM_ARSPD_CM, >0: Use this value in m/s instead of TRIM_ARSPD_CM.
+    // @DisplayName: Specific setting for airspeed when cruising while SOARING.
```

Comment:
Suggest "    // @DisplayName: Specific setting for airspeed when soaring in AUTO mode"

Reply:
Done